.github/workflows/build_and_test.yml

```diff
@@ -122,9 +132,9 @@ jobs:
       run: cd ./bindings/pylibafl && maturin build
 
   fuzzers:
-    env:
-      CC: ccache clang # use ccache in default
-      CXX: ccache clang # use ccache in default
+#     env: 
```

Comment:
maybe you can avoid disabling this globally but you can we overwrite this only in your build script

Reply:
in your case, build_nyx_support.sh

Reply:
`build_nyx_support.sh` should let the user decide which compile to use, I think. I add `CC=clang CXX=clang++` in `test_all_fuzzers.sh`

Reply:
> build_nyx_support.sh should let the user decide which compile to use, I think. I add CC=clang CXX=clang++ in test_all_fuzzers.sh
This will cause error, maybe due to the inconsistency of compile tools... So I just leave it as is

Reply:
does ccache really make a difference in CI time btw?

Reply:
The size of `ccache`'s cache is about 50MB, so just a little improvement

Reply:
Can' we increase that?

Reply:
no, the maximum limit I set for ccache is 2GB, but it only uses 50MB.
ccache is for c project, so it's limited to cache sub-projects like `libxml2` and `libpng`.

Reply:
Ah, I confused it with a sccache which also works for rust, my bad
https://doc.rust-lang.org/cargo/guide/build-cache.html#shared-cache